entrypoint.sh

```diff
@@ -159,11 +162,27 @@ if [[ -n "${MUMBLE_SUPERUSER_PASSWORD}" ]]; then
 fi
 
 # Show /data permissions, in case the user needs to match the mount point access
-echo "Running Mumble server as uid=$(id -u) gid=$(id -g)"
+echo "Entrypoint running as uid=$(id -u) gid=$(id -g)"
+echo "Preparing Mumble server to run as uid=$(MUMBLE_UID) gid=$(MUMBLE_GID)"
+
+if [[ "$MUMBLE_UID" != "0" ]]; then
+  groupmod -og "$MUMBLE_GID" mumble
+  usermod -ou "$MUMBLE_UID" mumble
+  if [[ "$MUMBLE_CHOWN" = true ]]; then
+    echo "Changing owner of folder {DATA_DIR}"
+    chown -R mumble:mumble ${DATA_DIR}
+  fi
+  exec runuser -u mumble -g mumble -- "$@"
```

Comment:
What does this do? If it starts the server, I don't think that should happen here (as well)

Reply:
indeed, it's a relica from a copy past

Reply:
removed with last commit